sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/config/Config.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 1999-2018 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alibaba.csp.sentinel.dashboard.config;
+
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.lang.math.NumberUtils;
+import org.springframework.lang.NonNull;
+
+/**
+ * Dashboard config support
+ * <p>
+ * Dashboard supports configuration loading by several ways by order:<br>
+ * 1. System.properties<br>
+ * 2. Env
+ * 
+ * @author jason
+ * @since 1.5.0
+ *
+ */
+public class Config {
```

Comment:
Maybe it's more clear to be `DashboardConfig` or others?

Reply:
I just think it is located in sentinel-dashboard and it's not a sharing library (totally final implementation). So creating an object named `DashboardConfig` in `Dashboard` is a little unnecessary? I am ok on calling it `DashboardConfig` or `Config` so let's just make a decision.

Reply:
Yes, it's okay to be `Config` or `DashboardConfig` or any other. I just thought it might be more convenient to search and identify in IDE when it's named as `DashboardConfig` :)

Reply:
OK then i will rename it.